examples/employment-based-loan/contracts/EmploymentLoan.sol

```diff
@@ -384,8 +385,34 @@ contract EmploymentLoan is SuperAppBase {
         bytes calldata, /*_agreementData*/
         bytes calldata, // _cbdata,
         bytes calldata ctx
-    ) external override returns (bytes memory newCtx) 
+    ) 
+        external 
+        override 
+        onlyHost
+        returns (bytes memory newCtx) 
     {
-        newCtx = _updateOutflow(ctx);
+        if (!_isCFAv1(_agreementClass)) {
+            newCtx = ctx;
+            return newCtx;
+        }
+        return _updateOutflow(ctx);
+    }
+
+    function _isCFAv1(address agreementClass) private view returns (bool) {
+        bytes32 CFA_ID = keccak256("org.superfluid-finance.agreements.ConstantFlowAgreement.v1");
```

Comment:
consider if makes sense having CFA_ID as constant variable, and not compute every time there is callbacks.